.github/workflows/aks-byocni.yaml

```diff
@@ -14,7 +14,7 @@ on:
   #   will disappear from the PR checks: please provide a direct link to the
   #   successful workflow run (can be found from Actions tab) in a comment.
   #
-  # pull_request: {}
+  pull_request: {}
```

Comment:
Should it be uncommented?

Reply:
https://github.com/cilium/cilium-cli/pull/1578/commits/fecd9d467c05c2c34953357a3aa510f9af751a7b will be removed before merging.